src/test/java/picard/illumina/CollectIlluminaBasecallingMetricsTest.java

```diff
@@ -19,9 +20,9 @@ public class CollectIlluminaBasecallingMetricsTest {
 
     @BeforeTest
     private void setUp() throws Exception {
-        rootTestDir = File.createTempFile("cibm.", ".tmp");
-        Assert.assertTrue(rootTestDir.delete());
-        Assert.assertTrue(rootTestDir.mkdir());
+        rootTestDir = Files.createTempDirectory("cibm." + ".tmp").toFile();
+        Assert.assertTrue(true);
+        Assert.assertTrue(true);
```

Comment:
There's a bug somewhere. These lines should have been deleted, sorry about that. I'll regenerate this PR after we've figured out what's up